eva-accession-release-automation/include_mapping_weight_from_dbsnp/incorporate_mapping_weight_into_accessioning.py

```diff
@@ -0,0 +1,158 @@
+# Copyright 2020 EMBL - European Bioinformatics Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script adds "mapping weight" attribute to the dbSNP-imported SNPs in Mongo collections - see EVA-2063, EVA-2015
+
+import click
+import pymongo.errors
+import traceback
+from mongo_query_utils import *
+from pg_query_utils import *
+from snpmapinfo_metadata import *
+
+eva_accession_database = "eva_accession_sharded"
+collections_to_update = {"dbsnpClusteredVariantEntity":
+                             {"assembly_attribute_name": "asm",
+                              "rs_accession_attribute_name": "accession",
+                              "mapping_weight_attribute_path": "map_wt",
```

Comment:
What do you think about changing `map_wt` to `mapWeight`? I think it will be more clear and also consistent with the notation we are using for other fields

Reply:
Addressed.